azure-pipelines.yml

```
@@ -47,6 +47,7 @@ jobs:
       - tokio-timer
       - tokio-trace
       - tokio-trace/tokio-trace-core
+      - tokio-trace/tokio-trace-core
```

I think this was a copy-paste mistake? I believe it needs to be
```suggestion
      - tokio-trace/test_static_max_level_features
```